api/src/pcapi/core/bookings/api.py

```diff
@@ -150,6 +159,11 @@ def book_offer(
     return individual_booking.booking
 
 
+def _venue_already_has_booking(venue_id: int) -> bool:
+
+    return Booking.query.filter(Booking.venueId == venue_id).first() is not None
```

Comment:
(détail / perf) Ta version fonctionne, mais elle récupère tous les champs de la table `booking` et SQLAlchemy construit un objet `Booking` dont on ne se sert pas. À la place, on pourrait utiliser `.exists()`  : 

```suggestion
    return db.session.query(Booking.query.filter(Booking.venueId == venue_id).exists()).scalar()
```

... et au passage supprimer la ligne vide inutile au début de la fonction. ;)

Reply:
Done